espnet/nets/pytorch_backend/e2e_asr_transformer.py

```diff
@@ -634,3 +634,74 @@ def calculate_all_attentions(self, xs_pad, ilens, ys_pad):
                 ret[name + "_time"] = m.attn_t.cpu().numpy()
                 ret[name + "_freq"] = m.attn_f.cpu().numpy()
         return ret
+
+    def ctc_align(self, x, y):
```

Comment:
How about moving this function to https://github.com/espnet/espnet/blob/master/espnet/nets/pytorch_backend/ctc.py or nets_utils.py?

Reply:
Thanks for the suggestion. I will try to move it to ctc.py.